Tests/Database/Realm/SchemaVersionTests.swift

```diff
@@ -0,0 +1,69 @@
+import XCTest
+@testable import Catalog
+import RealmSwift
+import Realm
+
+@available(iOS 14.0, *)
```

Comment:
Let's try to remove ios14 restriction if possible 😄

Reply:
Makes sense, done!